salt/modules/consul.py

```diff
@@ -2126,7 +2131,7 @@ def acl_delete(consul_url=None, **kwargs):
         ret['res'] = False
         return ret
 
-    function = 'acl/delete/{0}'.format(kwargs['id'])
+    function = 'acl/destroy/{0}'.format(kwargs['id'])
```

Comment:
Looking back at previous posts on Github, issues, commits, etc. for consul I see various references to the correct function being `destroy` rather than `delete`.  
Looking at this commit: 
https://github.com/hashicorp/consul/commit/5555e0eb9ba5a18a14a080ab6a08ba08ebf2af85

It has always been `destroy` and I used the wrong function when the code was initially added.  This change is safe to push forward.